test/bin/mirror_registry.sh

```diff
@@ -41,8 +34,21 @@ setup_registry() {
     # and it defaults to https. Since this is not supported we need to configure registries.conf so that skopeo tries http instead.
     sudo bash -c 'cat > /etc/containers/registries.conf.d/900-microshift-mirror.conf' << EOF
 [[registry]]
-location = "${REGISTRY_HOST}"
-insecure = true
+    prefix = ""
+    location = "${REGISTRY_HOST}"
+    insecure = true
+[[registry]]
+    prefix = ""
+    location = "quay.io"
+[[registry.mirror]]
+    location = "${REGISTRY_HOST}"
+    insecure = true
+[[registry]]
+    prefix = ""
+    location = "registry.redhat.io"
+[[registry.mirror]]
+    location = "${REGISTRY_HOST}"
+    insecure = true
 EOF
```

Comment:
Can we add some newlines so the sections (`registry` + its respective `mirror`) are grouped together?

```suggestion
[[registry]]
    prefix = ""
    location = "${REGISTRY_HOST}"
    insecure = true

[[registry]]
    prefix = ""
    location = "quay.io"
[[registry.mirror]]
    location = "${REGISTRY_HOST}"
    insecure = true

[[registry]]
    prefix = ""
    location = "registry.redhat.io"
[[registry.mirror]]
    location = "${REGISTRY_HOST}"
    insecure = true
EOF
```